airflow-core/src/airflow/ui/src/i18n/locales/de/assets.json

```
@@ -0,0 +1,29 @@
+{
+  "consumingDags": "Konsumierende Dags",
+  "createEvent": {
+    "button": "Ereignis erstellen",
+    "manual": {
+      "description": "Direkt ein Ereignis zu Datensets (Asset) erstellen",
```

Aligning regarding #51722
```suggestion
      "description": "Manuell ein Ereignis zu Datensets (Asset) erstellen",
```